Library/Formula/bazaar.rb

```diff
@@ -27,5 +28,23 @@ def install
     libexec.install 'bzr', 'bzrlib'
 
     bin.install_symlink libexec+'bzr'
+
+    # install the plugins under /var/bazaar/plugins/
+    plugins_orig = libexec/"bzrlib/plugins"
+    plugins_new = var/"bazaar/plugins"
+
+    Dir[plugins_orig/"*"].each do |plugin|
+      path = Pathname.new plugin
+      plugins_new.install plugin unless File.exist? (plugins_new/path.basename)
+    end
+
+    rm_rf plugins_orig
+    ln_s plugins_new, plugins_orig
+  end
```

Comment:
@mikemcquaid it worked with `-d`, but it didn’t install the plugin directory under `var`. Do you have an example of a good formula with `postinstall`? It’s sadly not in the [example formula](https://github.com/Homebrew/homebrew/blob/master/Library/Contributions/example-formula.rb).

I edited the PR to move the symlinks stuff in a `postinstall` method.